README.md

```diff
@@ -142,6 +143,24 @@ documentation](https://github.com/open-telemetry/opentelemetry-collector/blob/ma
 For troubleshooting information specific to this distribution, please review
 [troubleshooting.md](docs/troubleshooting.md).
 
+## Migrating from the SignalFx Smart Agent
+
+The Splunk distribution of OpenTelemetry Collector will be the next
+generation agent and gateway for Splunk APM and Splunk Infrastructure
+Monitoring.  As such, it is the replacement for the [SignalFx Smart
+Agent](https://github.com/signalfx/signalfx-agent).
+
+This distribution provides helpful components to assist current Smart Agent
+users in their transition to OpenTelemetry Collector and help ensure that
+no loss of functionality is experienced.  The [Smart Agent
+Receiver](./internal/receiver/smartagentreceiver/README.md), its associated
```

Comment:
Suggested change: "This distribution provides helpful components to assist current Smart Agent users in their transition to OpenTelemetry Collector and ensure no functionality loss."